article/app/controllers/ArticleController.scala

```diff
@@ -161,7 +161,7 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
 
   def renderArticle(path: String) = {
     LongCacheAction { implicit request =>
-      mapModel(path) {
+      mapModel(path, request.isEmail) {
```

Comment:
I mean rewriting the main (non-email) article templates – since this email one was more of a blank slate it seemed like a good place to start using cAPI blocks!